Library/Homebrew/cmd/update-report.rb

```diff
@@ -41,6 +48,7 @@ def update_report
         # Consider the message possibly missed if not a TTY.
         if $stdout.tty?
           safe_system "git", "config", "--local", "--replace-all", "homebrew.analyticsmessage", "true"
+          safe_system "git", "config", "--local", "--replace-all", "homebrew.cask.analyticsmessage", "true"
```

Comment:
I'd suggest `homebrew.caskanalyticsmessage` here

Reply:
Earlier ( https://github.com/Homebrew/brew/pull/4620#issuecomment-411760286 ) you suggested the current implementation. Could you confirm that you would now like it changed?